src/social-network/PostInfo.ts

```diff
@@ -39,13 +39,27 @@ export abstract class PostInfo {
     abstract readonly rootNodeProxy: DOMProxy
     // TODO: Implement this
     readonly postMetadataImages = new ObservableSet<HTMLImageElement>()
+    // The links appears under the post content
+    readonly postAttachments = new ObservableMap<string, PostInfoAttachment>()
```

Comment:
Would you like to move this into `postMetadataImages`?

Reply:
use other name?

maskbook file service, internal protocol name is `Attachment`